docs/manual/determinism-checker.tex

```diff
@@ -469,14 +469,34 @@
 \begin{Verbatim}
 @PolyDet("up") E remove(@PolyDet("noOrderNonDet") List<E> this, @PolyDet("use") int index);
 \end{Verbatim}
-This method signature disallows removing elements at specific indices from an  \<OrderNonDet List>.
+This method signature disallows removing elements at specific indices from an \<@OrderNonDet List>.
 Allowing such removals would be unsound. Consider the following example:
 \begin{Verbatim}
 @OrderNonDet List<@Det Integer> ondList;         // Suppose ondList contains elements: [1, 2, 3]
 ondList.remove(0);
 \end{Verbatim}
-Since the list is \<OrderNonDet>, removing an element at index 0 could remove any of its 3 elements in different runs
-making the resulting list \<NonDet> which is unsound.
+Since the list is \<@OrderNonDet>, removing an element at index 0 could remove any of its 3 elements in different runs
+making the resulting list \<@NonDet> which is unsound.
+
+\subsectionAndLabel{Type qualifier hierarchy among \<@PolyDet> variants}{determinism-poly-hierarchy}
+\begin{figure}
+    \begin{center}
+        \includeimage{poly-hierarchy}{6cm}
+    \end{center}
+    \caption{\<@PolyDet> type hierarchy}
+    \label{fig-poly-hierarchy}
+\end{figure}
+Figure~\ref{fig-poly-hierarchy} shows the type qualifier hierarchy among all the
+variants of \<@PolyDet>. Notice that \<@PolyDet> and \<@PolyDet("use")> are considered
+to be same in this hierarchy.
+Consider a method \code{m(@PolyDet ..., @PolyDet("use") ...)} that is overridden as
+\code{m(@PolyDet ..., @PolyDet("use") ...)}. Since \code{@PolyDet} is considered to be
```

Comment:
The two methods have the same signature; one of them should be changed, but I don't know which.